src/test/index.tsx

```diff
@@ -12,7 +12,6 @@ import { ThemeProvider } from 'theme'
 
 export type { RenderResult } from '@testing-library/react'
 export { act, waitFor } from '@testing-library/react'
-export { default as user } from '@testing-library/user-event'
```

Comment:
You could still `export { default as userEvent }` here, and then import from `.../test` and call `userEvent.setup()`. It may simplify imports in tests.